src/URI.h

```diff
@@ -6,53 +6,37 @@
 
 namespace uri {
 
-using std::array;
-using std::string;
-using std::vector;
-
 /// \brief Thin parser for URIs.
 class URI {
 public:
   /// \brief Creates a default URI with all members empty. You can (should) run
   /// URI::parse to fill it
-  URI();
+  URI() = default;
 
   /// Creates and parses the given URI
-  URI(string uri);
+  explicit URI(const std::string &URIString);
 
   /// Parses the given `uri`
-  void parse(string uri);
-
-  /// Given a `http://www.example.com` scheme will contain `http`.
-  std::string scheme;
+  void parse(const std::string &URIString);
 
   /// Parsed hostname.
-  std::string host;
+  std::string Host;
```

Comment:
Do we need both Host and HostPort?
Host seems to be used by the gelf stuff only. @dominikwerder could that take HostPort instead (assuming no port has been supplied, so it will still be the same value as Host)?

Reply:
That should work.